api/app/signals/apps/signals/migrations/0085_SIG-1708_category_changes.py

```diff
@@ -0,0 +1,163 @@
+from django.db import migrations
+
+SIG_1708_NEW_CATEGORIES = {
+    'schoon': {
+        'drijfvuil-bevaarbaar-water': {
+            'name': 'Drijfvuil bevaarbaar water',
+            'handling': 'I5DMC',
+            'slo': '5W',
+            'departments': [
+                'WAT',
+            ]
+        },
+        'veegzwerfvuil': {
+            'name': 'Veeg- / zwerfvuil',
+            'handling': 'A3DEC',
+            'slo': '3W',
+            'departments': [
+                'STW',
+            ]
+        },
+        'prullenbak-vol': {
+            'name': 'Prullenbak is vol',
+            'handling': 'A3DEC',
+            'slo': '3W',
+            'departments': [
+                'STW',
+            ]
+        },
+        'uitwerpselen': {
+            'name': 'Uitwerpselen',
+            'handling': 'A3WMC',
+            'slo': '21K',
+            'departments': [
+                'THO',
+                'STW',
+                'ASC',
+            ]
+        },
+        'gladheid-winterdienst': {
+            'name': 'Gladheid winterdienst',
+            'handling': 'GLADZC',
+            'slo': '3W',
+            'departments': [
+                'STW',
+            ]
+        },
+        'graffitiwildplak': {
```

Comment:
Vanwege consistentie zou je kunnen kiezen voor de key 'graffiti-wildplak', of staat deze vast vanwege externe systemen?

Reply:
ah, gezien regel 83 zal dit wel vast staan.